Thu Apr 14 10:39:38 EEST 2011  Sergei Trofimovich <slyfox@community.haskell.org>
  * added mingw32-gcc to MINGW search
diff -rN -u old-jhc/configure.ac new-jhc/configure.ac
--- old-jhc/configure.ac	2011-04-14 10:40:43.274101006 +0300
+++ new-jhc/configure.ac	2011-04-14 10:40:43.303101006 +0300
@@ -31,7 +31,7 @@
     echo [DrIFT not found: using preprocessed files]
 fi
 
-AC_CHECK_PROGS([MINGW],[mingw-gcc i386-mingw32-gcc i586-mingw32msvc-gcc],
+AC_CHECK_PROGS([MINGW],[mingw-gcc mingw32-gcc i386-mingw32-gcc i586-mingw32msvc-gcc],
                [i386-mingw32-gcc])
 
 GHC=$HC
